cpu/cc2538/radio/cc2538_rf.c

```diff
@@ -174,6 +174,11 @@ void cc2538_init(void)
     /* setup mac timer */
     _cc2538_setup_mac_timer();
 
+    /* Enable Auto ACK */
+    if (IS_USED(MODULE_IEEE802154_RADIO_HAL)) {
```

Comment:
I just noticed this module doesn't exists anymore... I will need to adapt it

Reply:
done. I simply removed it. The introduced configuration file can be used for both netdev and the Radio HAL